src/reducers/SelectedRevisions.tsx

```diff
@@ -0,0 +1,54 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
```

Comment:
I'm unsure why the linter didn't warn about this, but there should be a type here (see: https://github.com/mozilla/perfcompare/blob/master/src/types/state.ts#L24-L30). 

I will look into that.